### ISSUE TITLE
GCop 434

### ISSUE THREAD

**Issue by marjanjavid** [Geeksltd/GCop#155]
This rule is just usable in PropertyGetter. As the picture below show, there is no GCop hint for calling static methods in other places. If it is needed we should regard this too.
![434](https://user-images.githubusercontent.com/20684649/40573324-2b46373c-60d5-11e8-885c-09e8fe9095fc.png)

**Comment by Foroughi** [comment]
@marjanjavid  would you please be more specific? I don't follow on this issue

**Comment by marjanjavid** [comment]
Yes. please look at the first example of this document, which is added by @PaymonK  to this page.
https://github.com/Geeksltd/GCop/blob/master/Rules/GCop434.md

If we have this example, there would be no GCop hint for calling static methods(as you can see in the above picture line20).

So we must remove the Example1 or change the rule.

**Comment by Foroughi** [comment]
@marjanjavid  @Karvan I'm not sure if you want me to change GCop434 to cover the first example in [here](https://github.com/Geeksltd/GCop/blob/master/Rules/GCop434.md) . You may wanna check GCop433 as well. I think that example should be covered on GCop433, not GCop434

**Comment by Foroughi** [comment]
Also GCop433 will hint only if we call a static method (with Class name) in a **static** method, so here is the question: Why ?! why we need to just check that invocation only in static methods ?! I think by removing this (check only in static methods i mean) from GCop433, it will cover that first example as well

**Comment by marjanjavid** [comment]
I think it is better waiting @Karvan  or @PaymonK  answer our questions about these two rules.

**Comment by Karvan** [comment]
The screenshot is not clean @marjanjavid. it shows the VS pop up not GCop one. We don't need two rules for a single concept.
@Foroughi Could you please merge both into 433 and remove 434.

the logic: apart from the method or property. for calling a static method we don't need to call that by the class name.

@marjanjavid I'll update the rules documents.

**Comment by Foroughi** [comment]
@reporter @Karvan  Fixed on [NuGetPackages\a421f54d7b8059b6cd691606d9aae5234b4fb094](https://gitlab.com/Paymon/GCop/tree/a421f54d7b8059b6cd691606d9aae5234b4fb094)

Some notes : 

1- GCop 434 has been disabled 
2 - GCop 433 will kick in for any static method invocation (followed by the class name) regardless of where it's being called

**Comment by marjanjavid** [comment]
@Karvan I Send this screenshot to show that GCop434 is just kicked when we call a static method in Getter, not other places. In the other places VS recognize this problem not GCop.
